src/sentry/ingest/billing_metrics_consumer.py

```diff
@@ -206,7 +206,7 @@ def submit(self, message: Message[KafkaPayload]) -> None:
             future = track_outcome_custom_publish(
                 self._produce,
                 org_id=bucket_payload["org_id"],
-                project_id=bucket_payload["org_id"],
+                project_id=bucket_payload["project_id"],
```

Comment:
Whoops! Did a test case catch this?

Reply:
Yep, the test failed because the expected project id was `2` but the actual value was `1` instead.